style.css

```diff
@@ -419,14 +424,199 @@ input:focus {
   border-color: rgb(98, 108, 201);
 }
 
+.headerlist {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 5px;
+    background-color: white;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    list-style-type: none;
+    padding-right: 50px;
+    font-family: var(--primary-font);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    position: sticky;
+    top: 0;
+  }
+  
+  .headerlist a {
+    color: #344563;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
-    width: 80%;
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: cover;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border-bottom-left-radius: 300px 200px;
+    background-color: white;
+    background-position: center;
+  }
+
+  .main-Container {
+    width: 60%;
+    margin-left: 0;
+    padding-top: 0;
+  }
+
+  .work-container {
+    display: flex;
+    gap: 20px;
+    margin: 30px 10px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .cards {
+    width: auto;
+    flex-direction: row;
+    gap: 25px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .snapshot-portofolio img {
+    width: 100%;
+    height: auto;
+    padding-bottom: 0;
+  }
+
+  .blocks {
+    width: 40%;
+    gap: 10px;
+    height: auto;
+  }
+
+  .tframes {
+    gap: 12px;
+  }
+
+  .tframes h3 {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .frames-list {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .reverse-block {
+    flex-direction: row-reverse;
+    gap: 20px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .project-text {
+    align-items: center;
+    font-size: 16px;
```

Comment:
Hi @HussenYesufAlli 

Your work is excellent and wonderful, in most of the responsive sites, rem is used instead of px, it is good that you do this too.